.vscode/launch.json

```diff
@@ -54,7 +54,7 @@
             "name": "Launch Backend",
             "program": "${workspaceRoot}/examples/browser/src-gen/backend/main.js",
             "args": [
-                "--log-level=debug",
+                "--hostname=0.0.0.0",
```

Comment:
hello, I don't see how this change it is related to this feature ?

Reply:
we want to debug it in Gitpod